gum/arch-arm64/gumarm64writer.c

```diff
@@ -213,6 +213,7 @@ gum_arm64_writer_init (GumArm64Writer * writer,
   writer->ref_count = 1;
   writer->flush_on_destroy = TRUE;
 
+  writer->data_endian = __BYTE_ORDER__;
```

Comment:
Should use `G_BYTE_ORDER` and friends here and elsewhere.

Reply:
Fixed. Apologies for the delay.